### ISSUE TITLE
Does not work when superClass contains a dot

### ISSUE THREAD

**Issue by vbarbarosh** [suchipi/babel-plugin-transform-class-inherited-hook#1]
app.jsx

``` javascript
import React from 'react';

class App extends React.Component
{
}
```

$ babel app.jsx

```
TypeError: app.jsx: Cannot read property 'replace' of null
    at CodeGenerator.push (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/buffer.js:290:16)
    at CodeGenerator.Identifier (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/generators/types.js:37:8)
    at CodeGenerator._print (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/printer.js:159:17)
    at /usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/printer.js:95:13
    at CodeGenerator.withSource (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/buffer.js:273:5)
    at CodeGenerator.print (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/printer.js:94:10)
    at CodeGenerator.printJoin (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/printer.js:197:12)
    at CodeGenerator.printList (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/printer.js:262:17)
    at CodeGenerator.CallExpression (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/generators/expressions.js:143:8)
    at CodeGenerator._print (/usr/local/lib/node_modules/babel-cli/node_modules/babel-generator/lib/printer.js:159:17)
```

**Comment by vbarbarosh** [comment]
Replacing [this line](https://github.com/suchipi/babel-plugin-transform-class-inherited-hook/blob/master/src/index.js#L30) with

``` javascript
var PARENT = path.node.superClass;
```

will fix the issue.

**Comment by suchipi** [comment]
Thanks for the report. I will take a look at this sometime this week.

**Comment by suchipi** [comment]
Fixed in 3.1.1
